oemoflex/facades.py

```diff
@@ -516,6 +518,7 @@ def build_solph_components(self):
             {
                 self.bus: Flow(
                     nominal_value=self.capacity_turbine,
+                    variable_costs=self.marginal_cost,
```

Comment:
Was a bit confused to see this here. This happens because the branches of this PR and #91 are not clearly separated. Good for now, but better try to avoid in the future!

Reply:
This is due to a faulty branching again. Last two commits from #91 where repeated because of that. Sorry for that.